pastas/rfunc.py

```diff
@@ -851,7 +857,7 @@ def get_tmax(self, p, cutoff=None):
     def gain(self, p):
         return 1.
 
-    def step(self, p, dt=1, cutoff=None):
-        t = self.get_t(p, dt, cutoff)
+    def step(self, p, dt=1, cutoff=None, maxtmax=None):
```

Comment:
Mark this as an internal method just like the block?

Reply:
This is another file (rfunc.py instead of stressmodels.py)